doc/nrf/known_issues.rst

```diff
@@ -200,6 +200,13 @@ NCSDK-20047: SLM logging over RTT is not available
 Other issues
 ============
 
+.. rst-class:: v2-3-0
+
+CIA-892: Assert when printing strings with the ``%s`` qualifier
```

Comment:
This is assert or crash, I think. In ATv2 we saw a crash. I've seen an assert also.

Reply:
Correct, in ATv2 we saw crashes when using `printk()` to print large strings. handled in #10458 for ATv2, but the same may happen in other applications